scripts/changelog/Package.mjs

```diff
@@ -0,0 +1,37 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+export class Package {
+  /**
+   * @param {PackageDefinition} pkg
```

Comment:
This should be `PackageConfiguration`?

```suggestion
   * @param {PackageConfiguration} pkg
```

To be able to reference that, let's move it into this file from `changelog.mjs` instead:

```js
/**
 * @typedef {{
 *  path: string;
 *  excluded?: true;
 *  sameAs?: string;
 * }} PackageConfiguration A package in the monorepo to generate changelog for.
 */

export class Package {
  /**
   * @param {PackageConfiguration} pkg
   */
```

Then use it in `changelog.mjs`:

```js
/**
 * @type {import('./changelog/Package.mjs').PackageConfiguration[]}
 */
const PACKAGES = [
  { path: './packages/astro' },
  ...
```

Reply:
Oh this is nice! Thanks 😃